### ISSUE TITLE
Handlebars - `if` tags are broken when using helper with `textarea`

### ISSUE THREAD

**Issue by astronomersiva** [beautify-web/js-beautify#1482]
# Description
Ran this on a legacy codebase and ran into this issue whenever `textarea` is used.

# Input
The code looked like this before beautification:
```hbs
{{textarea value=someContent}}

{{#if condition}}
  <div class="some-class">{{helper "hello"}}<strong>{{helper "world"}}</strong></div>
{{/if}}
```

# Expected Output
The code should have looked like this after beautification:
```hbs
{{textarea value=someContent}}

{{#if condition}}
  <div class="some-class">{{helper "hello"}}<strong>{{helper "world"}}</strong></div>
{{/if}}
```

# Actual Output
The  code actually looked like this after beautification:
```
{{textarea value=tweetContent}}

{{#if conditioncondition
<div class="some-class">{{helper "hello"}}<strong>{{helper "world"}}</strong></div>
{{/if}}
```

# Steps to Reproduce

## Environment
OS: macOS
js-beautify: rc5

## Settings
Example:
```
js-beautify -r -s=2 --wrap_line_length=150 --type=html --indent_handlebars fileName
```

**Comment by bitwiseman** [comment]
Fixing ASAP.

**Comment by bitwiseman** [comment]
Fixed in and released in 1.8.0-rc6.
